spec/models/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe User do 
+
+	let!(:user) do
+    User.create(name: 'Dan Wheeler', username: 'd9nny',
+								email: 'dan@google.com', password: 'test123', 
+								password_confirmation: 'test123')
+  end
```

Comment:
This is seems like it vacuous - I am not sure what purpose this code has? Maybe I am missing something.